apps/cowswap-frontend/src/modules/permit/hooks/usePreGeneratedPermitInfo.ts

```diff
@@ -21,33 +21,9 @@ export function usePreGeneratedPermitInfo(): {
 
   const { data, isLoading } = useSWR(
```

Comment:
Linter errors here

Reply:
Yeah, that's for the temporary hack to make the PR testable.
Will revert the related change before merging.